pages/index.tsx

```diff
@@ -20,10 +22,58 @@ export default function Home(): JSX.Element {
     servicesList.taxonomies?.some(taxonomy => filters.includes(taxonomy)) ?? false
   const { taxonomyOptions, setFilters } = useTaxonomyFilter(baseServicesLists, setServicesLists, filterFunction)
 
-  const sortFieldsTextToVal = {Name: 'name', Description: 'description'}
+  const sortFieldsTextToVal = { Name: 'name', Description: 'description' }
 
   return (
-    <>
+    <Box w='100%' mb={16} display='flex' flexDirection='column'>
+
+      <Heading fontSize='5xl' textAlign={{ base: 'center', md: 'center' }} mt={8}>Services Lists</Heading>
```

Comment:
I feel like centering this heading looks weird?
<img width="1440" alt="Screen Shot 2022-01-17 at 9 20 16 PM" src="https://user-images.githubusercontent.com/54749942/149859806-1baf49f2-91fd-4523-aee7-3b5d10404124.png">

Reply:
Oh yeah I think it's because I messed up the merge for the page